manual/manual/macros.hva

```diff
@@ -49,9 +48,44 @@
   border: 1px solid grey;
   padding: 10px;
   overflow-y:auto;
-  white-space: pre-wrap;
+  display:flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+/*  white-space: pre-wrap;*/
 }
 
+
+
+\newstyle{.ocamlkeyword}{
+  font-weight:bold;
+}
+
+
+\newstyle{.ocamlhighlight}{
+  font-weight:bold;
+  text-decoration:underline;
+}
+
+\newstyle{.ocamlerror}{
+  font-weight:bold;
+  color:red;
+}
+
+\newstyle{.ocamlwarning}{
+  font-weight:bold;
+  color:purple;
+}
+
+\newstyle{.ocamlcomment}{
+  color:grey;
+}
+
+
+\newstyle{.ocamlstring}{
+  font-weight:bold;
```

Comment:
Sorry to nitpick, but emphasizing string content is a bit ugly (I would prefer to make it easier to visually ignore the string payload.) Could you maybe choose a color that is less de-emphasizing than "grey", but still doesn't make strings stand out? (A dark-enough blue or purple would do, depending on the rest of the style.)

Reply:
My understanding is that our color complexity budget is already quite strained, but it would be easy to use a lighter font or shade of green/red .

Reply:
Yes; whatever you like.

Reply:
Done with a bit of transparency.